docs/component_conventions.md

```diff
@@ -251,6 +251,46 @@ Print styles should be included in the main stylesheet for a component, using th
 }
 ```
 
+### Print style conventions
+
+- Ensure that any text colours are reset to black. Printed text colours can be set using the standard colour variable **$govuk-print-text-colour**. For example
+```color: $govuk-print-text-colour;```
+- Coloured borders should also be reset to black.
+- For inverted components, remove coloured backgrounds and reset text colors to black. If it's important to delineate the component from its surroundings, use a 1pt black border around the component:
+```border: 1pt solid $govuk-print-text-colour;```.
+- Some elments do not print by default in most browsers, background images for example. To force these to print as intended, use the CSS property print-color-adjust, for example:
+```
+  .gem-c-organisation-logo__container {
+    print-color-adjust: exact;
+    -webkit-print-color-adjust: exact;
+  }
+```
+
+### Print styles helpers
+
+There are currently three helper classes available for controlling print output. These are defined in `app/assets/stylesheets/govuk_publishing_components/lib/_print_support.scss`. These helper classes should be added to a component's view template in the appropriate places.
+
+#### Full page width columns
+
+**`.gem-print-columns-none`**
+
+A helper class for removing column layouts when printing. This should be applied to row/column layouts, by adding the class to elements with a `govuk-grid-row` class. The columns will then print at the full width of the page.
+
+Note that only the immediate child columns will be affected. If a child column also contains a row/column layout of further elements, these will be unaffected and will retain their existing column layout. Use the class again on these nested grids to apply fullwidth column printing if required.
+
+#### Consistent link formatting
+
+**`.gem-c-print-link`**
+
+This class ensures that printed links receive consistent formatting. It should be added to any link element that should be printed. Typically this  means applying it to links with the `govuk-link` class.
+
+Most links inherit styles from govuk-frontend, including the printing of the href that a link contains. This makes printed links more useful, but they can take up a lot of space on the page in the default font size and will print in blue by default. This class sets the printed colour to black, reduces the font size, and improves the layout of the printed href.
+
+**`.gem-c-print-links-within`**
+
+This is a variation of the previous class, to be used on _parent_ elements that contain links with the `govuk-link` class. This is only required when there's no way to add the `gem-print-link` directly in a component's view template (usually because the `govuk-link` element is not exposed to the view template as it's rendered from a self-contained helper.)
```

Comment:
Suggestion: `This can be used when the gem-print-link class cannot be applied directly to a link, for example when it is inside govspeak` - assuming that's correct?

Reply:
I've changed this to use the specific example where it was needed:

> This can be used when the `gem-c-force-print-link-styles` class cannot be applied directly to a link, such as with the [`inset_text` component](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_inset_text.html.erb).